### PR TITLE
change to typdef for rocblas_stride from typedef for rocblas_long

### PR DESCRIPTION
**Pull request by amcamd**
- a future PR will change to use rocblas_stride for stride arguments in rocblas_Xgemm_strided_batched

### CODE REVIEW

File: library/include/rocblas-functions.h [hunk 1]
```diff
@@ -2662,20 +2662,20 @@ ROCBLAS_EXPORT rocblas_status rocblas_gemm_strided_batched_ex(rocblas_handle
                                                               const void*       a,
                                                               rocblas_datatype  a_type,
                                                               rocblas_int       lda,
-                                                              rocblas_long      stride_a,
+                                                              rocblas_stride      stride_a,
                                                               const void*       b,
                                                               rocblas_datatype  b_type,
                                                               rocblas_int       ldb,
-                                                              rocblas_long      stride_b,
+                                                              rocblas_stride      stride_b,
```
Comment (daineAMD) [inline library/include/rocblas-functions.h]: Looks like the extra space is failing the format check.

File: library/src/blas3/rocblas_trtri_batched.hpp [hunk 2]
```diff
@@ -229,7 +229,7 @@ template <typename T>
 __global__ void rocblas_trtri_batched_fill(rocblas_handle handle,
                                            rocblas_fill   uplo,
                                            rocblas_int    n,
-                                           rocblas_long   num_zero_elem,
+                                           rocblas_stride   num_zero_elem,
```
Comment (daineAMD) [inline library/src/blas3/rocblas_trtri_batched.hpp]: Same here.

File: library/src/blas_ex/rocblas_gemm_ex.cpp [hunk 3]
```diff
@@ -392,20 +392,20 @@ extern "C" rocblas_status rocblas_gemm_strided_batched_ex(rocblas_handle    hand
                                                           const void*       a,
                                                           rocblas_datatype  a_type,
                                                           rocblas_int       lda,
-                                                          rocblas_long      stride_a,
+                                                          rocblas_stride      stride_a,
```
Comment (daineAMD) [inline library/src/blas_ex/rocblas_gemm_ex.cpp]: And here.

Comment (amcamd) [reply library/src/blas_ex/rocblas_gemm_ex.cpp]: Thanks, commit below should fix it.

**Comment by amcamd** [comment]
140 tests fail in calls to cblas_trtri on SLES. This is covered in SWDEV-203325, and it is not caused by this PR, so merging this PR.

**Pull request merged**